build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1334,6 +1334,17 @@ message ExecuteResponse {
 }
 
 // The current stage of action execution.
+//
+// Even though these stages are numbered according to the order in which
+// they generally occur, there is no requirement that the remote
+// execution system reports events along this order. For example, an
+// operation MAY to transition from the EXECUTING stage back to QUEUED
+// in case the hardware on which the operation executes fails.
```

Comment:
Maybe add "(although in such a case the system may instead choose to fail the operation and let the client retry)" to make it clear that automated retries in the event of h/w failure are not a requirement.

Reply:
I believe this is covered by using MAY.